pages/level/[id]/[slugName].tsx

```diff
@@ -90,23 +85,22 @@ interface LevelSWRProps {
 
 export default function LevelSWR({ level }: LevelSWRProps) {
   const router = useRouter();
-  const { id } = router.query;
-
-  if (router.isFallback || !id) {
+  if (router.isFallback || !level) {
     return <SkeletonPage/>;
   }
-
+  const id = level._id;
+  const levelProps = {level:level};
   return (
     <SWRConfig value={{ fallback: { [getSWRKey(`/api/level-by-id/${id}`)]: level } }}>
```

Comment:
Need to update this API endpoint (also if we move to slugs entirely with this PR we can remove this API and the associated hook as well)